cogs/verification.py

```diff
@@ -7,6 +7,8 @@
 import random
 from datetime import datetime
 
```

Comment:
This is being scrapped for a completely new system that I've been working on for a while now. So, I'll accept this but, in the grand scheme of things, it's not as relevant.